commonmark-extensions/src/Commonmark/Extensions/TaskList.hs

```diff
@@ -133,6 +133,17 @@ taskListItemBlockSpec = BlockSpec
                                listItemType lidata
                     -> addNodeToStack linode
                   _ -> addNodeToStack listnode >> addNodeToStack linode
+             blankAfterMarker <- optionMaybe lineEnd
+             pos' <- getPosition
+             case blankAfterMarker of
+                  Just _ -> return ()
+                  Nothing -> do
+                    toks <- many (satisfyTok (not . hasType LineEnd))
+                    addNodeToStack $
+                        Node (defBlockData paraSpec){
+                              blockStartPos = [pos']
+                            , blockLines = [toks] }
+                        []
```

Comment:
What happens if there are no tokens after the marker, or only spaces?  Do we get an empty Para?  Should that be avoided?

Reply:
There shouldn't be an empty para here. This code is part of a `case` statement that checks for that.